channels/stable-4.3.yaml

```diff
@@ -1,9 +1,27 @@
 name: stable-4.3
```

Comment:
I'm generally not in favor of backfilling (would you include all of stable-4.1 too?), and would rather folks who are this relaxed use version-agnostic channels (#112).  But however we do this, we don't want releases in a stable channel that are not in the corresponding fast channel, etc., so you'll want to add the new releases to `fast-4.3` and `candidate-4.3` as well.

Reply:
Yes, I think version agnostic channels would be preferred but that's too big of a change to make without more consideration from the architects team.

The main thing here is we have around 30 clusters in stable-4.3 and have for a few weeks because people've been anticipating a 4.3 upgrade any way now. Most of those clusters are not on 4.2.21 yet so they'd have to go back to stable-4.2 upgrade to 4.2.21 and switch back with out backfilling, correct?

Reply:
> Most of those clusters are not on 4.2.21 yet so they'd have to go back to stable-4.2 upgrade to 4.2.21 and switch back with out backfilling, correct?

Yup.  But we alert on `VersionNotFound`, right (I hope ;)?  If we don't, we should.  So folks interested in jumping to 4.3 from earlier 4.2 should have changed their channel to stable-4.3, immediately seen the alert and/or `RetrievedUpdates` go `False`, and then changed back to whatever they were on before.